hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java

```diff
@@ -72,6 +81,39 @@ public MapOperation(String name) {
         this.name = name;
     }
 
+    private int getRetryCount() {
+        HazelcastProperties properties = getNodeEngine().getProperties();
+        return properties.getInteger(FORCED_EVICTION_RETRY_COUNT);
+    }
+
+    private void runEvictionStrategies() {
+        final ILogger logger = this.logger();
+        final int forcedEvictionRetryCount = getRetryCount();
+
+        final Eviction[] evictions = new Eviction[] {
```

Comment:
This code can be on hot path under some conditions, so it is better not to create new objects for each run.

Reply:
I moved array initialization inside `MapOperation` constructor. However, I don't know if `Operation#run` method be invoked on multiple threads. `Eviction` implementations are stateful.